x-pack/platform/plugins/shared/spaces/public/plugin.tsx

```diff
@@ -188,5 +202,6 @@ export class SpacesPlugin implements Plugin<SpacesPluginSetup, SpacesPluginStart
       this.managementService.stop();
       this.managementService = undefined;
     }
+    this.spaceAndExecutionContextSyncSubscription.unsubscribe();
```

Comment:
I know it shouldn't happen, but this will break if for some reason somebody calls stop() without calling setup(), I'd rather do ` this.spaceAndExecutionContextSyncSubscription?.unsubscribe();` and remove the `!` from line 59.